src/main/java/com/cqse/teamscaleupload/TeamscaleUpload.java

```diff
@@ -26,9 +26,12 @@
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
 
 public class TeamscaleUpload {
 
+    public static final RequestBody EMPTY_BODY = RequestBody.create(null, new byte[0]);
```

Comment:
can you please resolve the TS findings?

Reply:
We can probably make some stuff private to resolve the interface comment warnings